packages/manager/apps/web/client/app/hosting/database/add/hosting-database-add.controller.js

```diff
@@ -12,9 +12,7 @@ angular
     'HostingDatabaseCreateCtrl',
     (
       $location,
-      $q,
```

Comment:
Those were unsused

Reply:
Nice catch, but it should be done in a different commit given the scope

Reply:
Done